bpf/init.sh

```diff
@@ -34,6 +34,9 @@ set -x
 # Enable JIT
 echo 1 > /proc/sys/net/core/bpf_jit_enable
 
+# Disable rp_filter
+echo 0 > /proc/sys/net/ipv4/conf/all/rp_filter
```

Comment:
Was commit 2cab586c499cfd424daa18aa11a1305691d25b28 not enough? What other interface needed disabling?